rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_native.cpp

```diff
@@ -1172,3 +1213,31 @@ void lv2_socket_native::set_non_blocking()
 	::fcntl(socket, F_SETFL, ::fcntl(socket, F_GETFL, 0) | O_NONBLOCK);
 #endif
 }
+
+bool lv2_socket_native::is_socket_connected()
+{
+	if (protocol != SYS_NET_IPPROTO_TCP || type != SYS_NET_SOCK_STREAM)
```

Comment:
Don't check protocol, SYS_NET_SOCK_STREAM will be TCP no matter what the protocol is.